examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -111,3 +113,85 @@
     fig.colorbar(axes[0].images[-1], cax=axes[-1])
     fig.suptitle("ERDS ({})".format(event))
     fig.show()
+
+##############################################################################
+# Similar as for :class:`~mne.Epochs` objects, we can also export data from
+# :class:`~mne.time_frequency.EpochsTFR` and
+# `~mne.time_frequency.AverageTFR` objects to a
+# :class:`Pandas DataFrame <pandas.DataFrame>`:
+
+df = tfr.to_data_frame()
+df.time = df.time / 1000  # time to s
+df.head()
+
+##############################################################################
+# This allows us to use additional plotting functions like
+# :func:`seaborn.lineplot` to easily plot confidence bands:
+
+freq_bands = {'delta': (0.5, 4),
+              'theta': (5, 7),
+              'alpha': (8, 14),
+              'beta': (15, 35)}
+fig, axes = plt.subplots(4, 3, sharey=True, sharex=True)
+for f_idx, f_band in enumerate(['beta', 'alpha', 'theta', 'delta']):
+    data = df.loc[(df.freq >= freq_bands[f_band][0]) &
+                  (df.freq <= freq_bands[f_band][1])]
+    for ch_idx, ch in enumerate(["C3", "Cz", "C4"]):
+        g = sns.lineplot(data=data.reset_index(),
```

Comment:
why do you need `.reset_index()` here?  `.to_data_frame()` should default to an integer index, no?

Reply:
yes, you're absolutely right. This is cruft.

Reply:
Done.